cross-chain/arbitrum/package.json

```
@@ -0,0 +1,76 @@
+{
+  "name": "cross-chain-arbitrum",
+  "version": "0.0.1",
+  "description": "Cross chain Arbitrum",
+  "license": "MIT",
```

We use GPL-3.0-only for core `solidity` package:
```suggestion
  "license": "GPL-3.0-only",
```

---

Bumping this up @dimpar @pdyraga

---

Let's use GPL-3.0-only everywhere.